tests/acceptance/ape-config.yaml

```diff
@@ -21,15 +21,15 @@ deployments:
   ethereum:
     local:
       - contract_type: NuCypherToken
-        address: 0  # account index
+        address: '0x1e59ce931B4CFea3fe4B875411e280e173cB7A9C'  # test account at index 0
```

Comment:
Fair question. The ape test accounts are generated based on a mnemonic in the config file. See [line 38](https://github.com/nucypher/nucypher/blob/f9b3babd69d9d1a93290ef53a4438e424a75d297/tests/acceptance/ape-config.yaml#L38). So the derived test accounts are always the same, unless that mnemonic changes - and I don't believe we have a reason to change it. 

The issue was that the newer version ape takes the "address" parameter and converts it to an actual address when pre-processing values, so "0" gets converted to "0x00000000000...00" address value which can't be converted to an int using `int(<value>)`, which then causes failures. My opinion here is that instead of using an index as the "address" value, because the test accounts are known, we can instead just use the test account addresses which are always the same anyway.

Reply:
got it, it makes sense. Thank you